pandas/tools/plotting.py

```diff
@@ -359,7 +359,10 @@ def scatter_matrix(frame, alpha=0.5, figsize=None, ax=None, grid=False,
     density_kwds = density_kwds or {}
 
     # workaround because `c='b'` is hardcoded in matplotlibs scatter method
-    kwds.setdefault('c', plt.rcParams['patch.facecolor'])
+    if 'color' in kwds:
```

Comment:
I meant, ``plt.scatter(..., facecolor='red')`` and ``plt.scatter(..., c='red')`` both draws points in red. So ``scatter_matrix(..., facecolor='red')`` and ``scatter_matrix(..., c='red')`` should be the same.